apps/daffio/src/app/docs/services/docs.service.ts

```diff
@@ -10,6 +10,8 @@ import { DaffioGuideList } from '../models/guide-list';
 import { DAFFIO_DOCS_PATH_TOKEN } from './docs-path.token';
 import { DaffioDocsServiceInterface } from './docs-service.interface';
 
+export const computeCrossEnvSafePath = (path: string) => path.replace(/[A-Z]/g, (a) => a.toLocaleLowerCase().concat('_'));
```

Comment:
Is there a good central place to export this so the logic isn't duplicated here and in dgeni?

Reply:
Moved to a new `docs-utils` package.